types.ts

```diff
@@ -1433,8 +1433,13 @@ export interface Skill {
   /**
    * Forces execution of a specific formula by name, overriding autonomous tool selection.
    * @hidden
-   * */
+   */
   forcedFormula?: string;
+  /**
+   * The LLM model to use for this skill. Specify a model name string.
```

Comment:
Where are developers going to learn about which model strings are valid? We should probably have a documentation page that lists the currently supported values and link to it. Don't have to do this in this PR, maybe @ekoleda-codaio can tackle that at his convenience.